style.css

```diff
@@ -0,0 +1,679 @@
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-right: 1rem;
+  }
+
+  .social ul li {
+    margin: 13px;
+  }
+
+  .social ul li a {
+    color: #eee;
+    text-decoration: none;
+  }
+
+  .item-bar {
+    background-color: #fff;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .nav-last ul {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: auto;
+  }
+
+  .nav-last ul li {
+    margin: 15px;
+  }
+
+  .nav-last .nav-last_img {
+    margin-right: 16rem;
+  }
+
+  .nav-last ul li a {
+    flex-direction: column;
+    color: #12161a;
+    text-decoration: none;
+  }
+
+  .nav-last ul li a:hover {
+    color: #ff0707;
+  }
+
+  .Fbutton {
+    border: 4px solid #fa4503;
+    padding: 0.8rem 1.6rem;
+  }
+
+  .nav-last .Fbutton:hover {
+    color: #ff0707;
+  }
+
+  .section1 {
+    background-image: url('./images/main_big.png');
+    background-position: 50% 50%;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-clip: content-box;
+    height: 100vh;
+  }
+
+  .welcome {
+    font-size: 5vh;
+    margin-left: 4rem;
+  }
+
+  .main-title {
+    font-size: 9vh;
+    margin-left: 4rem;
+  }
+
+  .paragragh1 {
+    width: 60%;
+    margin-left: 4rem;
+    padding: 0.5rem;
+    border: 2px solid white;
+    background-color: #7f8081;
+  }
+
+  .section1 footer {
+    margin-left: 4.5rem;
+  }
+
+  .main-programs {
+    margin-top: 0;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    flex-wrap: wrap;
+    height: auto;
+  }
+
+  .main-programs .program-parts {
+    display: flex;
+    flex-direction: row;
+    margin-left: 3rem;
+    width: 100%;
+  }
+
+  .main-programs h2 {
+    font-weight: 200;
+  }
+
+  .viewer {
+    background: none;
+    text-decoration: underline;
+    margin: 2rem;
+  }
+
+  .part {
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    width: 18%;
+    margin: -0.2rem;
+  }
+
+  .part:hover {
+    border: 1px solid white;
+  }
+
+  .program-parts h3 {
+    margin-top: 1rem;
+    font-size: 19px;
+    margin-right: 1rem;
+  }
+
+  .program-parts p {
+    margin-top: 1rem;
+    text-align: center;
+    font-weight: 100;
+  }
+
+  .artists h2 {
+    font-weight: 100;
+  }
+
+  #artist-section {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-left: 5rem;
+  }
+
+  .artist-section {
+    display: flex;
+    flex-basis: 40%;
+    margin: 2rem;
+  }
+
+  .artist-card {
+    display: flex;
+    flex-wrap: wrap;
+    flex-basis: 46%;
+  }
+
+  .second-heading {
+    font-size: 15px;
+    color: #fa4503;
+  }
+
+  .aboutme-heading {
+    color: #fa4503;
+    text-align: center;
+    font-size: 7vh;
+  }
+
+  .contact {
+    padding: 2rem;
+  }
+
+  .email {
+    padding: 1.2rem;
+  }
+
+  .aboutme-paragraph {
+    padding: 1rem;
+  }
+
+  .aboutme-line {
+    margin-top: 1.5rem;
+    width: 4%;
+  }
```

Comment:
- [ ] Kindly to solve this problem, that mentioned be the previous review, remove the `margin-top: 1.5rem;` here.

![image](https://user-images.githubusercontent.com/35535130/157782212-fbec35f7-e202-43c3-bf26-2e090d75b481.png)

Reply:
resolved